console_bridge.pc.in

```diff
@@ -1,8 +1,8 @@
 # This file was generated by CMake for @PROJECT_NAME@
 prefix=@CMAKE_INSTALL_PREFIX@
 exec_prefix=${prefix}
-libdir=${prefix}/lib
-includedir=${prefix}/include
+libdir=${prefix}/@CMAKE_INSTALL_LIBDIR@
```

Comment:
> Should I remove this part, or incorporate the changes from #29?

It only makes sense when using GNUInstallDirs, so probably the right way would to incorporate changes here and close pull request #29.